tests/shared/src/test/scala/org/http4s/PathSuite.scala

```diff
@@ -24,22 +24,8 @@ import org.scalacheck.Prop._
 class PathSuite extends Http4sSuite {
   checkAll("Order[Path]", OrderTests[Path].order)
   checkAll("Semigroup[Path]", SemigroupTests[Path].semigroup)
```

Comment:
Ah, we are already running tests for `Order[Path]`, then tests for `Eq[Path]` are redundant. Actually tests for `Hash[Path]` also run the `Eq` tests as well :)

Reply:
well spotted! Thanks :-)